types/setting.go

```diff
@@ -359,6 +362,25 @@ var (
 			string(VolumeAttachmentRecoveryPolicyImmediate),
 		},
 	}
+	SettingDefinitionNodeDownPodDeletionPolicy = SettingDefinition{
+		DisplayName: "Pod Deletion Policy When Node Downs ",
+		Description: "Defines the Longhorn action when a Volume is stuck with a StatefulSet/Deployment Pod on a down node.\n" +
+			"- **do-nothing** is the default Kubernetes behavior of never force delete StatefulSet/Deployment terminating pods.\n" +
+			"- **delete-statefulset-pod** Longhorn will force delete StatefulSet terminating pods on down node to release Longhorn volumes so that Kubernetes can spin up replacement pods.\n" +
```

Comment:
Thanks Joshua for the feedback! I think we need to use the word `force` because Kubernetes actually already did a graceful deletion of the pods when the Node Controller haven't received heartbeats form Kubelet for 5 mins and 40 seconds. After that the pod stay in terminating state and Longhorn needs to force delete the pod.

Reply:
I agree using the term `force` is more consistent and specific with our prior usage.

But  just to clarify my own understanding of the process:
- kube-controller requests deletion by setting the deletionTimestamp = (now + deletionGracePeriod)
- the terminating pod state is the result of having a set deletionTimestamp
- it's valid to reduce the deletionTimestamp (closer to now) but never allowed to increase it (further in the future)
- kubelet on the node is the one responsible for the actual deletion of the resource from the api-server
- kubelet will try to delete the resource once a deletionTimestamp is set, ie run finalizers, etc (that's what the grace period is for)
- kubelet must delete the resource after the deletionTimestamp has passed

Reply:
@joshimoo Awesome details! It absolutely correct! :D